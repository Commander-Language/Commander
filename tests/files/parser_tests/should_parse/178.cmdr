//Test precedence
x = a !=! a;