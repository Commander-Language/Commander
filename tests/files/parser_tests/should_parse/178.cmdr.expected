(PrgmNode
	(ExprStmtNode (BinOpExprNode (BindingNode x) SET (BinOpExprNode (LValueExprNode (VarExprNode a)) NOT_EQUAL (UnOpExprNode (LValueExprNode (VarExprNode a)) NOT)))))
